admin_manual/configuration_server/caching_configuration.rst

```diff
@@ -0,0 +1,193 @@
+==========================
+Configuring Memory Caching
+==========================
+
+You can significantly improve your ownCloud server performance with memory 
+caching, where frequently-requested objects are stored in memory for faster 
+retrieval. There are two types of caches to use: PHP opcache, and data caching 
+for your Web server. If you do not install a memcache you will see a warning on 
+your ownCloud admin page. (A memcache is not required and you may ignore the warning if you prefer.)
+
+A PHP opcache stores compiled PHP scripts so they don't need to be re-compiled 
+every time they are called. PHP bundles an opcache since version 5.5, so you 
+don't need to install or configure it.
```

Comment:
I thought that I need to enable it, but this depends on the distribution. On Arch for example they use the upstream default configuration: https://github.com/php/php-src/blob/ba2b0c11b33e00fb009f4a4d8fbf048e6473424a/php.ini-production#L1733

So maybe we should add that `opcache.enable` needs to be set to 1.